cfme/common/provider_views.py

```diff
@@ -580,7 +581,7 @@ def before_fill(self, values):
                 getattr(self, widget).fill(values.get(widget))
 
 
-class TemplatesCompareView(InfraProvidersView):
+class TemplatesCompareView(CompareView):
     """Compare Templates page."""
     title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
     comparison_table = Table(locator='//div[@id="compare-grid"]/table')
```

Comment:
This should be in the common comparison view as well

Reply:
Updated as suggested.